common/lazy-pages/Cargo.toml

```
@@ -0,0 +1,24 @@
+[package]
+name = "gear-common-lazy-pages"
```

```suggestion
name = "gear-lazy-pages-common"
```

Following our general naming (e.g. with backend crates)

---

https://github.com/gear-tech/gear/blob/4df202c4f76d6141b65e8fb0a8c277d76e01b6ce/common/codegen/Cargo.toml#L1-L3
https://github.com/gear-tech/gear/blob/4df202c4f76d6141b65e8fb0a8c277d76e01b6ce/core-backend/sandbox/Cargo.toml#L1-L3

---

> https://github.com/gear-tech/gear/blob/4df202c4f76d6141b65e8fb0a8c277d76e01b6ce/common/codegen/Cargo.toml#L1-L3

`crate-name` -> `crate-name-codegen` (helper module)
`gear-common` -> `gear-common-codegen`

> https://github.com/gear-tech/gear/blob/4df202c4f76d6141b65e8fb0a8c277d76e01b6ce/core-backend/sandbox/Cargo.toml#L1-L3

General topic: gear-backend
It's implementations: `gear-backend-sandbox` + `gear-backend-wasmi`
It's common part: `gear-backend-common`

https://github.com/gear-tech/gear/blob/4df202c4f76d6141b65e8fb0a8c277d76e01b6ce/core-backend/common/Cargo.toml#L1-L3

---

Lazy pages are inside common folder, so it's part of 'common', so must be 'common-lazy-pages'.

---

Ok rename as your want - not a big question